documentation/modules/con-entity-operator-logging.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-entity-operator.adoc
+
+[id='con-entity-operator-logging-{context}']
+= Operator loggers
+
+The Topic Operator and User Operator have a configurable logger:
+
+* `rootLogger.level`
+
+The operators use the Apache `log4j2` logger implementation.
+
+Use the `logging` property to configure loggers and logger levels.
```

Comment:
Suggested change:

Use the `logging` property in the `Kafka` resource to configure loggers and logger levels.

Reply:
Great idea! Done